backend/src/main/java/botobo/core/ui/auth/PathMethods.java

```diff
@@ -0,0 +1,19 @@
+package botobo.core.ui.auth;
+
+import java.util.List;
+
+public class PathMethods {
+    private final List<PathMethod> pathMethods;
```

Comment:
```suggestion
    private final Set<PathMethod> pathMethods;
```
중복으로 넣지 않아도 상관없으니 Set을 쓰는 게 어떨까요?
n이 작아서 거의 의미 없지만요..